library/styleguideComponents/ReactComponent/Utils/styles.css

```diff
@@ -94,6 +94,31 @@
     color: #576574;
     align-self: flex-start;
     text-align: justify;
+    overflow: hidden;
+    max-height: 4rem;
+}
+
+.react-rainbow-utils-item_description::before {
```

Comment:
I think this is not a good solution. We can use the`text-overflow: ellipsis` style

Reply:
The `text-overflow: ellipsis` work for a single line combined to `white-space: nowrap` but not for multiple line.